[1m============================= test session starts ==============================[0m
platform linux -- Python 3.6.6, pytest-3.6.0, py-1.7.0, pluggy-0.6.0 -- /usr/bin/python3
cachedir: .pytest_cache
driver: Remote
sensitiveurl: .*
metadata: {'Python': '3.6.6', 'Platform': 'Linux-4.15.0-20-generic-x86_64-with-Ubuntu-18.04-bionic', 'Packages': {'pytest': '3.6.0', 'py': '1.7.0', 'pluggy': '0.6.0'}, 'Plugins': {'xdist': '1.22.2', 'variables': '1.7.1', 'test-groups': '1.0.3', 'selenium': '1.13.0', 'rerunfailures': '4.1', 'repeat': '0.4.1', 'metadata': '1.7.0', 'html': '1.19.0', 'forked': '0.2', 'base-url': '1.4.1'}, 'Base URL': '', 'Driver': 'Remote', 'Capabilities': {'browserName': 'chrome'}, 'Server': 'selenium-hub:4444'}
rootdir: /opt/work, inifile: pytest.ini
plugins: xdist-1.22.2, variables-1.7.1, test-groups-1.0.3, selenium-1.13.0, rerunfailures-4.1, repeat-0.4.1, metadata-1.7.0, html-1.19.0, forked-0.2, base-url-1.4.1
[1mcollecting 0 items                                                             [0m[1mcollecting 0 items                                                             [0m[1mcollected 0 items                                                              [0m

------------------- generated xml file: /opt/work/result.xml -------------------
[33m[1m========================= no tests ran in 0.01 seconds =========================[0m
